tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/OpenTelemetrySdkTests.cs

```diff
@@ -298,6 +298,114 @@ await Verifier.Verify(resourceMetrics, settings)
                               .DisableRequireUniquePrefix();
             }
         }
+
+        [SkippableFact]
+        [Trait("Category", "EndToEnd")]
+        [Trait("RunOnWindows", "True")]
+        public void MeterListenerCapturesAllMetrics()
```

Comment:
This test doesn't invoke a test application, so I think this would be better placed in the `Datadog.Trace.Tests` project. There's a file called `ActivityTests.cs`, so perhaps this could be placed in a `MeterListenerTests.cs` file?